drivers/dai/intel/dmic/dmic_nhlt.c

```diff
@@ -344,11 +345,28 @@ int dai_dmic_set_config_nhlt(struct dai_intel_dmic *dmic, const void *bespoke_cf
 		return -EINVAL;
 	}
 
+	const uint32_t skip_regs[] = { (0x8 / 4), (0x10 / 4), (0x14 / 4), (0x18 / 4),
```

Comment:
Why is this done?

Reply:
I removed this part - latest change set MIC_MUTE bit for unused PDM registers only. Thanks